src/sentry/api/endpoints/project_plugins.py

```diff
@@ -13,6 +13,6 @@ def get(self, request, project):
         context = serialize([
             plugin
             for plugin in plugins.configurable_for_project(project, version=None)
-            if plugin.has_project_conf()
+            if plugin.has_project_conf() or plugin.get_plugin_type() == 'release-tracking'
```

Comment:
Yeah yeah, I'm saying we split the functionality up. Add a new check, and in Freight, you can return explicitly True/False however you want, but everyone else will fall back to `has_project_conf`.